policyengine_us/variables/gov/states/oh/tax/income/deductions/medical_exepenses/oh_uninsured_unreimbursed_medical_care_expenses.py

```diff
@@ -0,0 +1,35 @@
+from policyengine_us.model_api import *
+
+
+class oh_uninsured_unreimbursed_medical_care_expenses(Variable):
+    value_type = float
+    entity = Person
+    label = "Ohio unreimbursed medical and health care expense deduction for uninsured expenses"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2022/it1040-sd100-instruction-booklet.pdf#page=18",  # Line 36
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2022/it1040-sd100-instruction-booklet.pdf#page=27",
+        "https://codes.ohio.gov/ohio-revised-code/section-5747.01",  # R.C. 5747.01(10)
+    )
+    defined_for = StateCode.OH
+
+    def formula(person, period, parameters):
+        not_medicare_eligible = ~person("is_medicare_eligible", period)
+        employer_premium_contribution = person(
+            "employer_contribution_to_health_insurance_premiums_category",
+            period,
+        )
+        status = employer_premium_contribution.possible_values
+        health_insurance_premiums = person("health_insurance_premiums", period)
+        # Premiums only count if the employer paid none.
+        hipaid = select(
```

Comment:
remove this - also just note `hipaid` is an enum in the asec, not an informative/appropriate name for this variable which defines eligible premiums for this specific program

<img width="260" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/6076111/2036b5f8-db82-4613-8be9-a58bbe21c7d1">